packages/coreutils/selinux.patch

```diff
@@ -27,9 +29,52 @@
  
    /* FIXME: This handles new files.  But what about existing files?
       I.e., if updating a tree, new files would have the specified context,
---- coreutils-9.5.orig/src/mkdir.c
-+++ coreutils-9.5/src/mkdir.c
-@@ -228,7 +228,7 @@
+diff --git a/src/install.c b/src/install.c
+index b3b26abdb..c724dbe34 100644
+--- a/src/install.c
++++ b/src/install.c
+@@ -325,6 +325,9 @@ setdefaultfilecon (char const *file)
+   struct stat st;
+   char *scontext_raw = nullptr;
+ 
++  /* FIXME: Return early for now until a suitable workaround has been found */
```

Comment:
We may never find one, so how about:

`// Return early, before the selinux logic, because $reasons`

Reply:
Try this on for size.
```c
/* NOTE: Return early, before `install`'s selinux logic,
 * because we do not have suitable workaround for it */
```